tsconfig.json

```diff
@@ -14,7 +14,6 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "isolatedModules": true,
-    // "noEmit": true,

```

Comment:
https://www.conventionalcommits.org/en/v1.0.0-beta.3/
fix - when you want to fix